tests/module_cpp20/CMakeLists.txt

```diff
@@ -2,11 +2,21 @@ cmake_minimum_required(VERSION 3.28)
 
 project(json_test CXX)
 
-add_executable(json_test)
+set(nlohmann_json_DIR ${CMAKE_CURRENT_SOURCE_DIR}/../../build/modules)
```

Comment:
Isn't `build` a build folder whose name can be arbitrary? Please check and use `${CMAKE_BINARY_DIR}` instead.